test/functional/lua/buffer_updates_spec.lua

```diff
@@ -826,53 +826,53 @@ describe('lua: nvim_buf_attach on_bytes', function()
 
       feed("<esc>u")
       check_events {
-        { "test1", "bytes", 1, 8, 0, 0, 0, 0, 1, 1, 0, 4, 4 },
-        { "test1", "bytes", 1, 8, 0, 0, 0, 0, 4, 4, 0, 0, 0 }
+        { "test1", "bytes", 1, 9, 0, 0, 0, 0, 1, 1, 0, 4, 4 },
+        { "test1", "bytes", 1, 9, 0, 0, 0, 0, 4, 4, 0, 0, 0 }
```

Comment:
I just blindly changed these so that the tests pass, but I'll need some more time to prove to myself that these new values make sense.